packages/twenty-front/src/modules/views/hooks/internal/useViewFilters.ts

```diff
@@ -209,10 +209,11 @@ export const useViewFilters = (viewScopeId: string) => {
               return filtersDraft;
             }
 
-            if (filterToUpsert.value === '') {
-              filtersDraft.splice(existingFilterIndex, 1);
-              return filtersDraft;
-            }
+            // Fixes #2967: Keep the filter edition open if it is empty
```

Comment:
Using this kind of comment is generally considered a bad practice because git is already doing that job for us with blaming and proper commit messages. 

We can remove this code.